src/cargo/ops/cargo_add/mod.rs

```diff
@@ -559,6 +621,21 @@ fn get_latest_dependency(
     }
 }
 
+fn parse_rust_version(rust_version: impl AsRef<str>) -> (u64, u64, Option<u64>) {
```

Comment:
Should we simplify and do `unwrap_or(0)` inside this function?

Reply:
Yeah, that's probably fine.